packages/web3-constants/evm/token.json

```diff
@@ -899,6 +899,26 @@
         "Aurora": "",
         "Aurora_Testnet": ""
     },
+    "MDX_ADDRESS": {
```

Comment:
What does this field for?

Reply:
I will delete it, it will be necessary when heco was implemented.